src/Stryker.Core/Stryker.Core/MutantFilters/ExcludeFromCodeCoverageFilter.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.Mutants;
+using Stryker.Core.Options;
+using Stryker.Core.ProjectComponents;
+
+namespace Stryker.Core.MutantFilters
+{
+    public class ExcludeFromCodeCoverageFilter : IMutantFilter
+    {
+        string IMutantFilter.DisplayName { get; } = "exclude from code coverage filter";
+
+        IEnumerable<Mutant> IMutantFilter.FilterMutants(IEnumerable<Mutant> mutants, FileLeaf file, StrykerOptions options) =>
+            mutants.Where(m => !Exclude(m.Mutation.OriginalNode));
+
+        private static bool Exclude(SyntaxNode node) => 
```

Comment:
Could you refactor these methods to regular methods? We don't use arrow methods like this anywhere else in the codebase.

Reply:
Sure thing, but not entirely true: https://github.com/stryker-mutator/stryker-net/blob/cee97b5113e4896df1443fadf8a31bd7514ab08d/src/Stryker.Core/Stryker.Core/MutantFilters/ExcludeMutationMutantFilter.cs#L17

Reply:
Yeah true. But I think that is different from a whole expression that has to fit on another line 😉